### DISCUSSION TITLE
RG350P não inicia após atualizar, tela preta

### DISCUSSION THREAD

**Discussion by 4ilson** [eduardofilo/RG350_adam_image#271] [General]
após feita a atualização o console não inicia, apenas tela acesa, atalhos de brilho funcionando, reinicia, mais sem imagem, vários sd cards testados, varias versões diferentes e nada de iniciar!! ao regredir para a rom stock tudo flui perfeitamente!!

**Comment by eduardofilo** [comment]
First of all, for what you comment that the system in the background seems to work but only the display fails, are you sure you have correctly chosen the device model in the kernel selection script? There have been cases of people who thought they had an RG350P but factory installed the RG350M screen for example.

If that's not the problem, it's not the first case I've seen. There are some particular units that do not work well on ODbeta although they do work well on stock. Try to do a clean install of ODbeta to rule out that it's not something else, but I doubt it.

**Reply by 4ilson** [reply]
isso mesmo parece iniciar, mais não dá imagem, kernel selecionada corretamente RG350P,  rom stock RG350P funciona perfeitamente! usei todas as versões do ADAM e nenhuma funcionou!

**Reply by eduardofilo** [reply]
I can assure you that the image works on most RG350Ps. It must be a particular problem with your unit.
